Doc new deployment limit (#6756)
cloudflare/cloudflare-docs: content/pages/platform/limits.md

Builds will timeout after 20 minutes.

## Deployments

> Active deployments are limited to the most recent 10,000 excluding aliased deployments and production. Inactive deployments will return a 522 error and will need to be re-deployed. 

Reviewer: ```suggestion
Pages projects can have up to 10,000 active deployments. These are the most recent 10,000 deployments, excluding aliased deployments and production. Inactive deployments will return a `522` error and will need to be redeployed. All deployments are visible in the Cloudflare dashboard, but only the most recent 10,000 will be able to serve traffic.

```

> There is a limit of 10,000 active deployments, inactive deployments will return a 522 error and will need to be re-deployed. The latest aliased deployments are always active and so is production.

Reviewer 2: ```suggestion
Active deployments are limited to the most recent 10,000 excluding aliased deployments and production. Inactive deployments will return a 522 error and will need to be re-deployed. 
```
